### ISSUE TITLE
docs - add hint that doc-tests are supported

### ISSUE THREAD

**Issue by noah04** [rust-lang-nursery/lazy-static.rs#168]
The documentations [introductory section about the basic syntax](https://docs.rs/lazy_static/1.4.0/lazy_static/index.html#syntax) says:

> Attributes (including doc comments) are supported as well.

While reading it, I wondered if doc-tests are also supported. I tried it myself and it worked quite well. Is this an intended feature? If so, I'd like to create a PR to include that in the docs (maybe also with an example). 

Cheers and thanks for your work!

**Comment by LukasKalbertodt** [comment]
Yes, this is intended. Supporting doc comments kind of implies supporting doc tests. But I guess it doesn't hurt to mention it explicitly, so feel free to open a PR!
